app/importers/billing_contracts_import.rb

```diff
@@ -0,0 +1,69 @@
+class BillingContractsImport
+  include ActiveModel::Model
+  require 'roo'
+
+  attr_accessor :file
+
+  def initialize(attributes = {})
+    attributes.each { |name, value| send("#{name}=", value) } if !attributes.nil? && attributes.key?(:file)
+  end
+
+  def perform
+    if !file
+      errors.add :base, 'Field can\'t be empty!'
+      false
+    else
+      finding_errors
```

Comment:
no need for this check
as with current flow without file this code would not be executed